### PR TITLE
updated installation readme with correct API URL

### PR DESCRIPTION
**Pull request by jeffthorne**
Updated running_grafeas.md based on Arun's suggestion in the google group.
https://groups.google.com/forum/#!topic/grafeas-users/ZdnByaqaPik

Thanks for the help. Saved me a lot of time :-)

Cheers,
Jeff

### CODE REVIEW

**Comment by jeffthorne** [comment]
I signed it!

File: v1alpha1/proto/grafeas.proto [hunk 1]
```diff
@@ -28,7 +28,7 @@ import "protoc-gen-swagger/options/annotations.proto";
 option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
   info: {
     title: "An API to insert and retrieve metadata on cloud artifacts.";
-    version: "v1alpha1";
+    version: "v1beta1";
```
Comment (vtsao) [inline v1alpha1/proto/grafeas.proto]: Please revert this, this is the alpha proto, the beta protos are here:
https://github.com/grafeas/grafeas/tree/master/proto/v1beta1

File: v1alpha1/proto/grafeas.swagger.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "An API to insert and retrieve metadata on cloud artifacts.",
-    "version": "v1alpha1"
+    "version": "v1beta1"
```
Comment (vtsao) [inline v1alpha1/proto/grafeas.swagger.json]: Same as above, please revert this.

**Comment by aysylu** [comment]
Superceded by #238, thanks for catching the issues.

**Pull request closed without merging**